src-react/redux/reducers/addons.reducer.js

```diff
@@ -13,6 +13,13 @@ const nameList = (state = initialStateNameList, action) => {
         newState.push(action.data.name)
       }
       return newState
+    case 'delAddonObj':
+      newState = [...state]
+      let idx = newState.indexOf(action.data.name)
```

Comment:
youre not reAssigning idx value so it should be safe to use `const`

Reply:
fixed